### ISSUE TITLE
PdbScriptsPipelineRunCommand.java is too big

### ISSUE THREAD

**Issue by sheridancbio** [cBioPortal/pdb-annotation#33]
I would separate all the "construct a command" type of functions to a utility class (or classes). I would create separate classes to handle:
    - run a local process functions (runwithRedirect)
    - downloadfile functionality
    - blast processing functions (makedb, blastp)
    - gunzip functionality
    - ftp file download and parsing
That would leave the main run methods (run, runInit, runUpdatePDB) .. which might also be split up unless they share some common code. The ReadConfig class can/should be used in many places to provide access to the properties.

**Comment by juexinwang** [comment]
Yes, I think I have several common functions. I can improve it as you mentioned
